### ISSUE TITLE
Scaled up processes remain in stopping even after scale back down

### ISSUE THREAD

**Issue by temujin9** [flynn/flynn#3922]
```
$ flynn -c prod-us-east-1 -a prod-bvjs-apps-scoutfile ps --all
ID                                                  TYPE    STATE     CREATED         RELEASE
ip1012065146-92a837ed-a16e-4795-8a19-cfccb3d07005   run     down      8 days ago      6a2b9082-23ef-483a-9c15-2738cc31de3e
ip10120141170-808b4c9f-ac74-4135-b8fe-06487790eb24  dimsum  down      8 days ago      8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip10120201230-81749b9c-713f-445c-9ba4-34de96684ad4  web     down      8 days ago      8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip1012065146-9ff5d0db-803a-4e4c-989f-2ca8fa7daca5   dimsum  down      7 days ago      8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip10120201230-67f4f2f0-1317-4963-9c88-d2eaf0f507de  dimsum  down      7 days ago      8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip1012065146-90d55a5f-e7bb-4a3d-867b-4e4b178f7c04   dimsum  down      7 days ago      8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip10120141170-3275a830-334f-4e51-801a-3e81c16d88b8  dimsum  down      7 days ago      8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip10120141170-01371445-baa1-42ab-a1a8-976d4c0afead  dimsum  down      7 days ago      8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip10120141170-c355ea4f-7ed4-4667-8962-29c0335fc51e  dimsum  down      7 days ago      8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip1012065146-0b05d886-ab3b-4c01-8fb9-608534bcb48d   dimsum  down      7 days ago      8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip10120201230-8b2582a7-c41f-410d-a653-bbb40c539225  dimsum  down      7 days ago      8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip10120201230-251ff90c-6521-46c0-8d10-6ac7b5be5288  dimsum  down      7 days ago      8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip10120194201-bb700268-af7f-48f4-923e-9ed5e8d738e9  web     down      47 hours ago    8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip10120194201-66ea15cc-a074-4925-b6db-420a6d832388  dimsum  down      47 hours ago    8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip1012076118-1e26cc7f-07f0-4d9d-94ff-d267db8cdf71   web     stopping  25 hours ago    8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip1012093225-433ff6fc-bd22-4585-a219-ba70efdc7cf0   dimsum  stopping  25 hours ago    8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip10120114181-8984081a-ca11-45de-b409-f1caca305fd8  dimsum  down      16 minutes ago  8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip1012076118-1ce6026b-5413-448b-93e4-558c1ad82d8c   dimsum  stopping  12 minutes ago  8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip1012076118-77533fd4-61aa-484b-86b9-b68553bfae17   dimsum  stopping  11 minutes ago  8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip1012093225-b2c6ed2b-5b1d-4dc9-83a8-1cf5a2f4313d   dimsum  stopping  11 minutes ago  8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip10120114181-05783325-dd3c-4e9e-929f-c243409c6119  dimsum  down      11 minutes ago  8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip1012076118-cb168f02-4407-40d4-9f2f-e4a1a2dcdb36   dimsum  stopping  11 minutes ago  8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip1012093225-2896f2aa-2540-4efd-bbae-0542d8cf430a   dimsum  stopping  11 minutes ago  8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip1012093225-039bc3f6-2116-4abb-b3bc-8254d85a8b5e   dimsum  stopping  11 minutes ago  8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip1012093225-88058b57-b52b-45f0-83ae-b8b626acdfbc   dimsum  stopping  11 minutes ago  8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b
ip10120114181-3cda027d-9a62-4f70-8b7a-3924281c5c6a  dimsum  down      11 minutes ago  8cfa9519-c0d8-48fb-b75e-eb3f2d13a57b

$ flynn -c prod-us-east-1 -a prod-bvjs-apps-scoutfile kill ip1012076118-1e26cc7f-07f0-4d9d-94ff-d267db8cdf71 
ERROR: could not kill job ip1012076118-1e26cc7f-07f0-4d9d-94ff-d267db8cdf71: unknown_error: Something went wrong
Could not kill all jobs.
```

This happened after a user scaled a process up from 1 instance to 10, to attempt handling some excessive load, and then scaled back down to 0 instances.

Debug info incoming soon.

**Comment by temujin9** [comment]
https://bazaarvoice.box.com/s/ybvgymu9q58h9inuq7344t4gffhpqa5q is the debug info.

**Comment by lmars** [comment]
I'm pretty sure this was due to the host not accepting requests to stop the jobs due to the fd leak fixed in #3928.

Closing (please open a new issue if this happens again).
